clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -78,6 +78,13 @@ public class IngestionStats {
   private final Int2ObjectMap<Sensor> regionIdToHybridBytesConsumedSensorMap;
   private final Int2ObjectMap<Sensor> regionIdToHybridRecordsConsumedSensorMap;
   private final Int2ObjectMap<Sensor> regionIdToHybridAvgConsumedOffsetSensorMap;
+
+  // write path latency sensors
+  private final WritePathLatencySensor producerSourceBrokerLatencySensor;
+  private final WritePathLatencySensor sourceBrokerLeaderConsumerLatencySensor;
+  private final WritePathLatencySensor producerLocalBrokerLatencySensor;
+  private final WritePathLatencySensor localBrokerFollowerConsumerLatencySensor;
+  private final WritePathLatencySensor leaderProducerCompletionLatencySensor;
```

Comment:
I think it makes sense to have those here... that's better organized than having them inside DIV stats... thanks.